Dockerfile

```diff
@@ -2,6 +2,16 @@
 
 FROM golang:1.17.5-alpine3.15
 
+# Build arguments.
+#
+# Override on the docker command line with
+#   --build-arg USE_TSL_SOCKETS=false
+#   --build-arg ENABLE_HOST_CHECK=false
+#
+ARG USE_TSL_SOCKETS=true
```

Comment:
Shouldn't this be TLS_SOCKETS?

Reply:
Thank you. You are definitely correct. I have updated that commit to remove that typo.